yarn.lock

```diff
@@ -1972,6 +1972,21 @@
     "@griffel/react" "^1.0.0"
     tslib "^2.1.0"
 
+"@fluentui/react-infobutton@9.0.0-beta.99":
+  version "9.0.0-beta.99"
+  resolved "https://registry.yarnpkg.com/@fluentui/react-infobutton/-/react-infobutton-9.0.0-beta.99.tgz#d58c8816ce1bb61c0d5b8515a75971c07d2a75e8"
+  integrity sha512-egrYTSoOvbvWl3aYAgU1M6sBPJbsxeSQuqXpbZvn0AXH99AhMOjVLsiDUkolNWRbbxp5V2zh22Q/1Hg0bSp+Og==
+  dependencies:
+    "@fluentui/react-icons" "^2.0.224"
+    "@fluentui/react-jsx-runtime" "^9.0.35"
+    "@fluentui/react-label" "^9.1.67"
+    "@fluentui/react-popover" "^9.9.3"
+    "@fluentui/react-tabster" "^9.19.6"
+    "@fluentui/react-theme" "^9.1.19"
+    "@fluentui/react-utilities" "^9.18.6"
+    "@griffel/react" "^1.5.14"
```

Comment:
ATM this dedupes nicely , but this will change with any major bump of those packages. 

probably more brainstorming should be put into this how to approach this
- my immediate idea would be to completely template the `unstable/index.ts` file and add these unstable dependencies only during pre-publish step during release